config/initializers/dfe_analytics.rb

```diff
@@ -54,13 +54,15 @@
   # use a new version of the BigQuery streaming APIs.
   config.azure_federated_auth = true
 
+  if Rails.env.in?(%w[development review qa])
+    # Path of airbyte stream config file relative to the App root (Rails.root)
+    config.airbyte_stream_config_path = "terraform/aks/workspace-variables/airbyte_stream_config.json"
+  end
```

Comment:
```suggestion
  # Path of airbyte stream config file relative to the App root (Rails.root)
  config.airbyte_stream_config_path = "terraform/aks/workspace-variables/airbyte_stream_config.json"
  
  config.airbyte_enabled = true if Rails.env.development?
```
The ENV variable `BIGQUERY_AIRBYTE_DATASET` may not always be set in development, depending on individual developer setup. If not set then the check for the Airbyte stream config will be omitted.  

Setting the `airbyte_stream_config_path` by itself and not enabling airbyte will not do anything.

Reply:
thanks @asatwal. I'm not sure I fully understand. All I was trying to do was make it possible to run the `dfe:analytics:regenerate_airbyte_stream_config` rake task rather than enable Airbyte in dev.

Reply:
Tested the suggestion locally and it works for me.

Reply:
So enabling airbyte won't actually do anything at run time apart from checking the static airbyte config ie `airbyte_stream_config.json` on server start up.

The static airbyte config is used at deployment time, If airbyte is enabled for Terraform also.

So setting the `airbyte_stream_config_path` allows you to generate the airbyte config in the right location and with corrcect filename.

Enabling airbyte in development checks the static airbyte config against the database on startup to ensure that it is up to date with the schema.